### PR TITLE
🐛  DEVEP-2353: ActionButtons should open git urls in separate tab

### PR DESCRIPTION
**Pull request by macdonst**
## Description

Don't use JS to navigate to a new page, use a link instead.

## Related Issue

DEVEP-2353

## Motivation and Context

Properly open external issues and github links in a new tab

## How Has This Been Tested?

- Storybook
- gatsby develop/build/serve

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/ActionButtons/index.js [hunk 1]
```diff
@@ -9,37 +9,49 @@ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTA
 OF ANY KIND, either express or implied. See the License for the specific language
 governing permissions and limitations under the License.
 */
-import React from 'react'
```
Comment (sic2) [inline src/ActionButtons/index.js:12]: I am not familiar with `@emotion`. What is the benefit of it? 
IMHO, having simple React makes the code easier to read, maintain and more solid

File: src/ActionButtons/index.js [hunk 2]
```diff
@@ -9,37 +9,49 @@ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTA
...
 
-const ActionButtons = ({ gitUrl, filePath, branch, issues, ...props }) => (
+const ActionLink = ({ href, children }) => (
+  <a
+    href={href}
+    css={css`
+      text-decoration: none;
+    `}
+    target='_blank'
+    rel='noopener noreferrer nofollow'
```
Comment (sic2) [inline src/ActionButtons/index.js:27]: ⭐

File: src/ActionButtons/index.js [hunk 3]
```diff
@@ -9,37 +9,49 @@ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTA
...
-        issues
-          ? (document.location.href = issues)
-          : (document.location.href = `${gitUrl}/issues/new?body=Issue%20in%20${filePath}`)
-      }}
-      aria-label='Log issue'
+    <ActionLink href={`${gitUrl}/edit/${branch}/${filePath}`}>
+      <ActionButton isQuiet aria-label='Edit page' excludeFromTabOrder>
+        <Edit size='S' />
+        <Text>Edit this page</Text>
+      </ActionButton>
```
Comment (sic2) [inline src/ActionButtons/index.js:45]: Should we not expect a change in any snapshot tests?

**Review by sic2 — APPROVED** [review]
Approved. 
However, I think it would be good to update the docs README for this component. 
Nothing complex, just an example usage for the ActionButton. 
Would be good to start writing the docs ;)

File: src/ActionButtons/index.js (hunk 1 above)
Comment (macdonst) [reply src/ActionButtons/index.js:12]: @emotion allows us to do CSS in JS a lot easier.

File: src/ActionButtons/index.js (hunk 3 above)
Comment (macdonst) [reply src/ActionButtons/index.js:45]: We need more and better tests. Right now everything passes. Someday, I'd love to go through this repo and do nothing but add tests and docs.

**Pull request merged**